spec/unit/installer/analyzer_spec.rb

```diff
@@ -672,84 +672,30 @@ module Pod
 
       #-------------------------------------------------------------------------#
 
-      it 'handles test only pod targets' do
```

Comment:
tests are deleted because `pod_target_test_only?` is deleted. Similar tests moved to `resolver_spec.rb`